src/ontology/foodon-edit.owl

```diff
@@ -18460,6 +18460,125 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003933 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003933">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003928"/>
+        <obo:IAO_0000115 xml:lang="en">A pizza that is topped with a combination of four kinds of cheese melted together, with or without tomato sauce.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-09T14:22:17Z</dc:date>
+        <oboInOwl:hasExactSynonym xml:lang="en">4 cheese pizza</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">four cheese pizza</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003934 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003934">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003928"/>
+        <obo:IAO_0000115 xml:lang="en">A pizza with pesto sauce, which is made from basil, pine nuts, oil, garlic and Parmesan cheese, along with other pizza toppings.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 rdf:datatype="http://www.w3.org/2000/01/rdf-schema#anyURI">https://en.wikipedia.org/wiki/Pesto</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-09T16:30:17Z</dc:date>
+        <rdfs:label xml:lang="en">pesto pizza</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003935 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003935">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003934"/>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03311484"/>
+        <obo:IAO_0000115 xml:lang="en">A pesto pizza made with chicken meat.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-09T16:31:00Z</dc:date>
+        <rdfs:label xml:lang="en">chicken pesto pizza</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003936 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003936">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003934"/>
+        <obo:IAO_0000115 xml:lang="en">A pesto pizza made with vegetables.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-09T16:31:07Z</dc:date>
+        <oboInOwl:hasExactSynonym xml:lang="en">veggie pesto pizza</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">vegetable pesto pizza</rdfs:label>
```

Comment:
I think generally we don't say "vegetable pizza", but rather "vegetarian pizza".